templates/land-see/land-see.amp.html

```diff
@@ -117,18 +117,18 @@
     <amp-animation id="subscribeBGParallax" layout="nodisplay">
       <script type="application/json">
         {
-        "duration": "1s",
-        "fill": "both",
-        "direction": "alternate",
-        "animations": [
-          {
-            "selector": ".land-see-subscribe-bg",
-            "keyframes": [
-              { "transform": "translateY(calc(height('.land-see-subscribe') * -0.4))" }
-            ]
-          }
-        ]
-      }
+          "duration": "1s",
```

Comment:
remove tabs and replace with spaces.

Reply:
Hmm... those should be spaces. I double checked by showing invisibles in Atom. They appear to be showing up as spaces, there. I have "soft" tabs enabled, too, which should substitute each tab for 2 spaces.

![screen shot 2017-10-18 at 8 46 36 pm](https://user-images.githubusercontent.com/523157/31750429-a9a084d0-b445-11e7-89b2-28bf5bffb4b1.png)